### PR TITLE
Define error metadata tags

### PR DESCRIPTION
**Pull request by izumin5210**
### What does this PR do?
Add error metadata tags to `ext/tags.js` defined in https://docs.datadoghq.com/tracing/trace_explorer/trace_view/?tab=spantags#more-information

### CODE REVIEW

File: ext/tags.js [hunk 1]
```diff
@@ -9,6 +9,9 @@ const tags = {
   SAMPLING_PRIORITY: 'sampling.priority',
   ANALYTICS: '_dd1.sr.eausr',
   ERROR: 'error',
+  ERROR_MSG: 'error.message',
```
Comment (izumin5210) [inline ext/tags.js:12]: :memo: dd-trace-go changes `error.msg` to `error.messages` on 2022-12-07
https://github.com/DataDog/dd-trace-go/pull/1605

Comment (rochdev) [reply ext/tags.js:12]: What is your use case for setting these manually? It should be enough to set the `ERROR` tag to an error object which will then automatically be converted to the appropriate tags internally. Setting these manually is discouraged.

**Review by rochdev — CHANGES_REQUESTED** [review]

**Comment by tlhunter** [comment]
@izumin5210 it looks like Roch had some open concerns about this PR. Is it something you'd still like to work on? If not I can close it for now.

**Comment by tlhunter** [comment]
I'll close this PR for now but we can consider reopening it in the future.

**Pull request closed without merging**